src/doc/python_scripting/functions.rst

```diff
@@ -4369,17 +4369,15 @@ GetQueryOutputObject
   GetQueryOutputObject() -> dictionary or value
 
 return type : dictionary or value
-    GetQueryOutputObject returns an xml string produced by the last query.
+    GetQueryOutputObject returns a dictionary or value.
 
 
 **Description:**
 
-    Both the GetQueryOutputString and GetQueryOutputValue functions return
-    information about the last query to be executed but the type of information
-    returns differs. GetQueryOutputString returns a string containing the
-    output of the last query. GetQueryOutputValue returns a single number or
-    tuple of numbers, depending on the nature of the last query to be executed.
-    GetQueryOutputXML and GetQueryOutputObject expose more complex query output.
+    GetQueryOutputObject, GetQueryOutputString, GetQueryOutputValue and
+    GetQueryOutputXML all return output from the last query to be executed
+    but with the type of information returned. GetQueryOutputObject returns
```

Comment:
Do we not use sent. per line in these `.rst` files because they are used to gen python doc strings? Does python not handling line wrapping in its `help()` output?

Not sure what is meant by "...but with the type of information returned."